install_files/appstream/io.github.openrazer.openrazer.metainfo.xml

```diff
@@ -141,6 +141,7 @@
     <modalias>usb:v1532p026Ad*</modalias>
     <modalias>usb:v1532p026Dd*</modalias>
     <modalias>usb:v1532p0276d*</modalias>
+    <modalias>usb:v1532p0270d*</modalias>
```

Comment:
Regenerate this file with `scripts/generate_appstream_file.sh`

Reply:
@z3ntu  @BeanieBen9990  Made a pull request for this change.